apps/emqx_ai_completion/src/emqx_ai_completion_schema.erl

```diff
@@ -65,6 +65,13 @@ fields(provider) ->
                 default => openai, required => true, desc => ?DESC(type)
             })},
         {api_key, emqx_schema_secret:mk(#{required => true, desc => ?DESC(api_key)})},
+        {base_url,
+            mk(binary(), #{
+                required => false,
+                desc => ?DESC(base_url),
+                validator => fun emqx_schema:non_empty_string/1,
+                default => <<"https://api.openai.com/v1">>
```

Comment:
Q: will this default work with `anthropic`?

Reply:
I thought that making union type for providers is an overkill so far. Let's expect that it will be obvious for a user to change endpoint if they change type.